server/actorController.js

```diff
@@ -39,21 +38,23 @@ const actorController = {
   },
 
   getActorCharacters: (req, res, next) => {
```

Comment:
Can we combine this with the getCharacterData middleware? Have the endpoint always return the full character data but filter it down to a specific actor when the actor query parameter is set?

Reply:
Sidenote: Chaining middleware where multiple functions are doing "business logic" stuff can make things hard to follow or reason about.   
  
With chained business logic functions, it may be unclear where specific functionality belongs. Devs might not even notice that there are multiple places it could be.  
  
I might suggest having one middleware function do the actual work while any extra middleware functions do more generic things like transforming data from one format to another, checking authorization, caching stuff, etc.

Reply:
Good tip, thank you! Fixed in new commit.